dashboard/pages/data_projection/page.py

```diff
@@ -0,0 +1,25 @@
+from dash import register_page, html, dcc
+
+from dashboard.pages.builders import ProcessPageBuilder
+from dashboard.pages.data_projection.stages import STAGES
+from dashboard.pages.data_projection.callbacks import register_callbacks
+from dashboard.storage.local import LocalFileStorage
+
+
+NAME = "Data Projection"
+register_page(path="/data-projection", name=NAME, module=__name__)
+
+pb = ProcessPageBuilder(name=NAME)
+STAGE_NAMES = [
+    "Files Input",
+    "Projections Visualization",
```

Comment:
I would name it just Visualization, since the stage name already consists  "projection"

Reply:
Good point, done 👍